google/earthenterprise
# Removed timedLock before getting task list.

Resolves #1483 

Since `GetCurrTasks` is just reading information, and since it doesn't really matter if the task list is slightly out of date for a few seconds, it's not really necessary to acquire a lock before before reading. We just need to make sure we catch any exceptions that could occur.

**Verification steps**

- Create a project or database that requires at least a few minutes of "load and think" time initially.
- Start the build and run `getop` while the system manager is busy
- Verify that `getop` is able to display information
- After system manager "unlocks" and starts creating tasks, verify that `getop` is able to show the running tasks without causing any errors during the build
